README.md

```diff
@@ -227,18 +230,26 @@ $recommendations = $response->getRecommendation()->getData();
 // }
 ```
 
+You can further modify which items will be reccomended by providing boosting rules. Priority of items matching the
+MQL `$criteria` will be multiplied by the value of `multiplier`:
+
+```php
+$reccomendation = UserRecommendation::create('user-id', 'test-scenario')
+    ->addBoost(Boost::create('valid_to >= NOW()', 2))
+    ->addBoost(Boost::create('for_recommendation = 1', 3.5));
+```
+
 #### Recommendation response properties
 
 Every item in Matej has its id, and optionally other item properties. These properties can be set up in [item properties setup](#item-properties-setup-to-setup-you-matej-database),
 and you can upload item data in the [events](#send-events-data-to-matej) request. This has major benefit because you can request
 these properties to be returned as part of your Recommendation Request.
 
-We call them response properties, and they can be specified either as the last parameter of `UserRecommendation::create` function,
-by calling `->addResponseProperty()` method, or by calling `->setResponseProperties()` method. Following will request an `item_id`,
+We call them response properties. They can be specified by calling `->addResponseProperty()` method or by calling `->setResponseProperties()` method. Following will request an `item_id`,
 `item_url` and `item_title`:
 
 ```php
-$recommendation = UserRecommendation::create('user-id', 5, 'test-scenario', 1.0, 3600, ['item_url'])
+$recommendation = UserRecommendation::create('user-id', 'test-scenario')
```

Comment:
It is not clear why this will return "`item_id`, `item_url` and `item_title`" as stated in the paragraph above. How do you achieve such behavior (ie. returning item_url)?

Reply:
It was a mistake in the example code. Btw. we will always return item_id and technically it's not a item property. We are still not sure what should matej API do if user requests it. But that's out of scope of this PR.